src/main/java/org/entur/lamassu/model/entities/MultiPolygon.java

```diff
@@ -58,6 +69,27 @@ public static MultiPolygon fromGeoJson(org.geojson.MultiPolygon geometry) {
       )
       .collect(Collectors.toList());
     mapped.setCoordinates(mappedCoordinates);
+
+    var mappedEncodedCoordinates = coordinates
+      .stream()
+      .map(polygon ->
+        polygon
+          .stream()
+          .map(ring ->
+            ring
+              .stream()
+              .map(lngLatAlt ->
+                Point.fromLngLat(lngLatAlt.getLatitude(), lngLatAlt.getLongitude()) // because GeoJson is defined with [lon, lat] instead of [lat, lon] used by PolylineUtils.encode, the order is switched on purpose here
+              )
+              .collect(Collectors.toList())
+          )
+          .map(ring -> PolylineUtils.encode(ring, 5))
```

Comment:
Could be considered to set the precision to 6 instead.
https://stackoverflow.com/questions/1947481/how-many-significant-digits-should-i-store-in-my-database-for-a-gps-coordinate

Reply:
What is the size implications? Precision of 5 means error margin of 2 meters, which could be significant in terms of riding and parking restrictions.

Reply:
Updated to 6 now. Note that this is intended mainly for graphical purposes in the app. To validate a parking location, exact data should be used server side.